wiki/Home.md

```diff
@@ -21,26 +22,54 @@ You can find here information about the organization and the workflow of this re
 
 ## Pull request
 
-- Each PR must be linked to the corresponding issue. 
+- Each PR must be linked to the corresponding issue.
 - You can use the Draft PR, if you want an early review
-- For the reviewer, you can prefix your comments with a keyword that indicating the kind of comment it is, here is a short list of keywords : 
-   - Important
-   - Nitpick
-   - Question
-   - Bug
-   - Proposition
+- For the reviewer, you can prefix your comments with a keyword that indicating the kind of comment
+  it is, here is a short list of keywords :
+    - Important
+    - Nitpick
+    - Question
+    - Bug
+    - Proposition
+
+## Gradle
+
+Make sure to have gradle tasks in the gradle tab inside Android studio.
+
+![ExpectedGradle](image/ExpectedGradle.png)
+
+If your tab isn't like the one on the screen shot, go to `File->Settings...->Experimental`,
+inside `Gradle` section, uncheck `Do not build Gradle task list during Gradle sync`
```

Comment:
Proposition: This step alone didn't help to show Gradle tasks tab even after I reloading the IDE. I had to re-build the project. Maybe it's worth to mentioning.